src/Containers/BoxComposition2/BoxComposition2.tsx

```diff
@@ -0,0 +1,115 @@
+import React,{useState,useEffect} from 'react'
+import styled from 'styled-components'
+
+interface IBoxProps{
+    imgSrc:string;
+    top:number;
+    left:number;
+    width:number;
+    height:number;
+    borderRadius:number;
+}
+
+export interface IBoxComposition2Props{
+    box1:IBoxProps;
+    box2:IBoxProps;
+    box3:IBoxProps;
+}
+
+export const BoxComposition2:React.FC<IBoxComposition2Props>=({box1,box2,box3}):React.ReactElement=>{
+    const [showBox2,setShowBox2]=useState(false)
+    const [showBox3,setShowBox3]=useState(false)
+    useEffect(()=>{
+        setTimeout(()=>{
+            setShowBox2(true)
+        },1000)
+        setTimeout(()=>{
+            setShowBox3(true)
+        },2000)
+    },[])
+    return     (
+        <div>
+            <Box1 {...box1} />
+            {showBox2&&<Box2 {...box2} />}
+            {showBox3&&<Box3 {...box3} />}
+        </div>
+    )
+}
+
+const Box3:React.FC<IBoxProps>=({imgSrc,top,left,width,height,borderRadius}):React.ReactElement=>
+    (
+        <AnimatedDiv3 top={top} left={left}>
+            <Img src={imgSrc} width={width} height={height} borderRadius={borderRadius} />
+        </AnimatedDiv3>
+    )
+
+const Box2:React.FC<IBoxProps>=({imgSrc,top,left,width,height,borderRadius}):React.ReactElement=>
+    (
+        <AnimatedDiv2 top={top} left={left}>
+            <Img src={imgSrc} width={width} height={height} borderRadius={borderRadius} />
+        </AnimatedDiv2>
+    )
+
+const Box1:React.FC<IBoxProps>=({imgSrc,top,left,width,height,borderRadius}):React.ReactElement=>
+    (
+        <AnimatedDiv1 top={top} left={left}>
+            <Img src={imgSrc} width={width} height={height} borderRadius={borderRadius} />
+        </AnimatedDiv1>
+    )
+
+interface IDivProps{
```

Comment:
Can't this be replaced with HTMLDivProps?

Reply:
no, I don't think so...